.circleci/config.yml

```diff
@@ -77,4 +77,4 @@ workflows:
   multiple-ghcs:
     jobs:
       - stackage-lts22
-      - stackage-nightly
+      - stackage-lts23
```

Comment:
the current stackage nightly is now using ghc-9.10.1.

So I renamed the job to lts-23 to reflect reality, but now repo settings is expecting stackage-nightly job that doesn't exist.

To fix this we can:
- bump nightly resolver to use ghc 9.10 (more work - I would prefer not doing that in this PR)
- change repo settings not to expect stackage-nightly job as mandatory, but to expect stackage-lts23 (but I don't have permissions to fix repo settings)

Reply:
I'll fix the repo settings